media/js/base/global-init.js

```diff
@@ -11,5 +12,11 @@ $(document).ready(function() {
     initLangSwitcher();
     $(window).on('load', function () {
         $('html').addClass('loaded');
+
+        /* Bug 1264843: In partner distribution of desktop Firefox, switch the
+           downloads to corresponding partner build of Firefox for Android. */
+        if (Mozilla.Client.isFirefoxDesktop) {
```

Comment:
It just occurred to me that this might also be fine to call outside of the `load` event, inside the `ready` handler above.